LLVMFloatToFixed/IndirectCallConversion.cpp

```diff
@@ -0,0 +1,78 @@
+#include "LLVMFloatToFixedPass.h"
+#include "TypeUtils.h"
+#include "llvm/ADT/SmallPtrSet.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Intrinsics.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Support/raw_ostream.h"
+#include <llvm/Transforms/Utils/Cloning.h>
+#include <llvm/Transforms/Utils/ValueMapper.h>
```

Comment:
fix include style

Reply:
move llvm includes to <>

Reply:
The majority of includes across all TAFFO passes uses double quotes for llvm headers, should we use <> here as suggested or convert our two <> to double quotes?

Reply:
IMHO <> should be more appropriate. However, I'll leave the final decision to @danielecattaneo

Reply:
I moved all the LLVM imports to double quotes as suggested by @danielecattaneo in the Initializer PR.